src/steps/index.test.ts

```diff
@@ -1,92 +1,107 @@
-      },
-      required: ['manager'],
-    },
-  });
+    // Act
+    await fetchCorps(context);
+    await fetchUsers(context);
+
+    // Assert
+    expect({
+      numCollectedEntities: context.jobState.collectedEntities.length,
+      numCollectedRelationships: context.jobState.collectedRelationships.length,
+      collectedEntities: context.jobState.collectedEntities,
+      collectedRelationships: context.jobState.collectedRelationships,
+      encounteredTypes: context.jobState.encounteredTypes,
+    }).toMatchSnapshot();
 
-  const users = context.jobState.collectedEntities.filter((e) =>
-    e._class.includes('User'),
-  );
-  expect(users.length).toBeGreaterThan(0);
-  expect(users).toMatchGraphObjectSchema({
-    _class: ['User'],
-    schema: {
-      additionalProperties: false,
-      properties: {
-        _type: { const: 'acme_user' },
-        firstName: { type: 'string' },
-        _rawData: {
-          type: 'array',
-          items: { type: 'object' },
+    const accounts = context.jobState.collectedEntities.filter((e) =>
+      e._class.includes('Organization'),
+    );
+    expect(accounts.length).toBeGreaterThan(0);
+    expect(accounts).toMatchGraphObjectSchema({
+      _class: ['Organization'],
+      schema: {
+        additionalProperties: false,
+        properties: {
+          _type: { const: 'sigsci_corp' },
+          _rawData: {
+            type: 'array',
+            items: { type: 'object' },
+          },
+          name: { type: 'string' },
+          displayName: { type: 'string' },
+          createdOn: { type: 'number' },
+          smallIconURI: { type: 'string' },
+          siteLimit: { type: 'number' },
+          sitesUri: { type: 'string' },
+          authType: { type: 'string' },
+          logoutURI: { type: 'string' },
+          samlCert: { type: 'string' },
+          samlEndpoint: { type: 'string' },
+          signRequestsUsingStoredCert: { type: 'boolean' },
+          samlRequestCert: { type: 'string' },
+          sessionMaxAgeDashboard: { type: 'number' },
+          apiTokenMaxAge: { type: 'number' },
+          restrictedAccessTokens: { type: 'boolean' },
+          ssoProvisioningConfigured: { type: 'boolean' },
         },
       },
-      required: ['firstName'],
-    },
-  });
+    });
 
-  const userGroups = context.jobState.collectedEntities.filter((e) =>
-    e._class.includes('UserGroup'),
-  );
-  expect(userGroups.length).toBeGreaterThan(0);
-  expect(userGroups).toMatchGraphObjectSchema({
-    _class: ['UserGroup'],
-    schema: {
-      additionalProperties: false,
-      properties: {
-        _type: { const: 'acme_group' },
-        logoLink: {
-          type: 'string',
-          // Validate that the `logoLink` property has a URL format
-          format: 'url',
-        },
-        _rawData: {
-          type: 'array',
-          items: { type: 'object' },
+    const users = context.jobState.collectedEntities.filter((e) =>
+      e._class.includes('User'),
```

Comment:
I would go ahead and do strict equality. Same for above

Reply:
I gave your suggestion a go, but it appears that _class is actually an array of strings.

Reply:
I mentioned elsewhere that we can avoid the `.filter()` here, but FWIW, if filtering, it'd be better to match on `e._type === 'sigsci_<type>'` explicitly